packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -19,6 +19,17 @@ const INTERVAL_SECONDS = 1
 // Maximum attempts to retrieve a balance
 const MAX_ATTEMPTS = 20
 
+export const DEFAULT_MEMO = {
+  data: 'xrpl-js',
+  type: 'user-agent',
+  format: 'text',
+}
+interface memo {
+  data?: string
+  type?: string
+  format?: string
+}
```

Comment:
I think this is missing the "Memo" field. 
Ex. "Memos": [
        {
            "Memo": {
                "MemoType": "687474703a2f2f6578616d706c652e636f6d2f6d656d6f2f67656e65726963",
                "MemoData": "72656e74"
            }
        }
    ],
```suggestion
interface memo {
  Memo: {
    data?: string
    type?: string
    format?: string
  }
}
```

Reply:
This type can be imported from `src/models`

Reply:
> This type can be imported from `src/models`

the models is wrong - faucet API takes in another format

Reply:
actually this fix needs more fixing - I think the type is wrong

Reply:
i'll figure it out later today